.github/workflows/build.yml

```diff
@@ -28,33 +22,35 @@ jobs:
       - name: Update Fastlane
         run: bundle update fastlane
 
+        # Required for KotlinMultiplatform
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '11'
+
       - name: Pod Install
         run: bundle exec fastlane cru_shared_lane_pod_install
 
       - name: Run Tests
         run: bundle exec fastlane cru_shared_lane_run_tests
 
       - name: Download And Commit Latest OneSky Localizations
-        if: github.event_name == 'workflow_dispatch'
+        if: github.event_name == 'push' && github.ref == 'refs/heads/master'
         run: bundle exec fastlane cru_shared_lane_download_and_commit_latest_one_sky_localizations
 
-      - name: Decode AppleAppStoreApi.json.enc For Fastlane App Store Connect Access 
-        if: github.event_name == 'workflow_dispatch'
+      - name: Decrypt AppleAppStoreApi.json.enc For Fastlane App Store Connect Access 
+        if: github.event_name == 'push' && github.ref == 'refs/heads/master'
         env: 
           KEY: ${{ secrets.ENCRYPTED_CF3A694B4ADE_KEY }}
           IV: ${{ secrets.ENCRYPTED_CF3A694B4ADE_IV }}
         run: openssl aes-256-cbc -K ${KEY} -iv ${IV} -in fastlane/AppleAppStoreApi.json.enc -out fastlane/AppleAppStoreApi.json -d
```

Comment:
If the API key is an actual json file, we can just store the contents of that file as a secret instead of having to use openssl to decode the file

Reply:
Hey @frett I like this idea better, I will look into this.

Reply:
we've done it for some Android projects, let me get you a link to a sample

Reply:
https://github.com/CruGlobal/godtools-android/blob/develop/.github/workflows/build.yml#L115-L118

Reply:
We can generate a new App Store API key from their UI so you don't have to worry about extracting the current one. let me know if you need help with this or can't access it.

Reply:
@frett I think I can use this instead https://docs.fastlane.tools/actions/app_store_connect_api_key/

How would we go about generating a new App Store API key and using that?

Reply:
that is what the `AppleAppStoreApi.json` key is ;) it's found on app store connect under "Users and Access" -> "Keys"